qiime2/core/cite.py

```diff
@@ -0,0 +1,68 @@
+import os
```

Comment:
Please double check the [lint-trap](https://travis-ci.org/qiime2/qiime2/builds/363300593)!